.gitignore

```diff
@@ -10,6 +10,7 @@ lib-cov
 pids
 logs
 results
+tags
```

Comment:
Please don't mess with the .gitignore just cause on your system certain extra files are generated.